### PR TITLE
Update requirements.txt for future updates

### PR DESCRIPTION
**Pull request by Taewan-P**
### Summary

Since the latest version of aiohttp is 3.6.2, it says that it is incompatible with discord.py's latest version, 1.2.4. So I am suggesting an update on requirements.txt for future updates.

### Checklist

- [ ] If code changes were made then they have been tested.
    - [ ] I have updated the documentation to reflect the changes.
- [ ] This PR fixes an issue.
- [ ] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [x] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

**Comment by Taewan-P** [comment]
Also attaching this [commit](https://github.com/Rapptz/discord.py/commit/e2478b17c998bb56fd96e183ddac053ba66363ae). It seems that he committed the aiohttps version prior to the update for future updates because aiohttp 3.6.0 was released on September, 2019, but as you see the commit date is Feb. I thought it was a good idea, so I updated one too.

**Comment by Gelbpunkt** [comment]
I am using aiohttp 3.6.2 in several bots without any issues. +1 for this, dependencies should stay up to date

**Comment by iomintz** [comment]
aiohttp.web from aiohttp 3.5 is also broken in python3.8 when using GunicornWebWorker. In my case I run a web server that imports my bot code, so I'm using aiohttp 3.5. I therefore can't use 3.8 for my web server.

**Comment by RDIL** [comment]
@iomintz just out of interest, why did you choose that approach?

**Comment by iomintz** [comment]
The API server performs the exact same functions as the bot, so I wrote my bot functions against bot.http and called them from the API server in order to minimize duplicate code.

File: requirements.txt [hunk 1]
```diff
@@ -1,2 +1,2 @@
-aiohttp>=3.3.0,<3.6.0
+aiohttp>=3.5.4,<4.0.0
```
Comment (Rapptz) [inline requirements.txt]:
```suggestion
aiohttp>=3.6.0,<3.7.0
```

Comment (Taewan-P) [reply requirements.txt]: I just changed it...!

Comment (Taewan-P) [reply requirements.txt]: @Rapptz Is it good?

**Pull request closed without merging**